### PR TITLE
[IMP] accounting: checks payment

### PR DESCRIPTION
**Pull request by chiaraprattico**
taskid: 3062532

### CODE REVIEW

File: content/applications/finance/accounting/payables/pay/check.rst [hunk 1]
```diff
@@ -2,39 +2,28 @@
 Pay by checks
 =============
 
-Once you decide to pay a supplier bill, you can select to pay by check.
-Then, at the end of the day, the manager can print all checks by batch.
-Finally, the bank reconciliation process will match the checks you sent
-to suppliers with actual bank statements.
+Once you decide to pay a supplier bill, you can select to pay by check. You can then print all the
+the registered payment by checks. Finally, the bank reconciliation process will match the checks you
+sent to suppliers with actual bank statements.
```
Comment (jaae-odoo) [inline content/applications/finance/accounting/payables/pay/check.rst:7]:
```suggestion
Once you decide to pay a supplier bill, you can select to pay by check. You can then print all the
registered payments by check. Finally, the bank reconciliation process will match the checks you
sent to suppliers with actual bank statements.
```

File: content/applications/finance/accounting/payables/pay/check.rst [hunk 2]
```diff
@@ -2,39 +2,28 @@
...
-screen you get when you enter the accounting application), click on your bank
-account on :menuselection:`More --> Settings` option. On the
-**Payment Method** field, set **Check**.
+The first necessary step to pay by checks is to activate the setting. To do so go to
+:menuselection:`Accounting app --> Configuration --> Checks`, there you can activate the payment
+method as well as setup the :guilabel:`Check Layout`.
+Once you have activated the setting, you must also activate the payment method on related bank
+journals. Go to :menuselection:`Accounting dashboard --> Bank Journal --> drop-down menu (⋮) -->
+Configuration`.
+In the :guilabel:`Outgoing Payments` tab, add :guilabel:`Checks` as a payment method.
```
Comment (jaae-odoo) [inline content/applications/finance/accounting/payables/pay/check.rst:21]:
```suggestion
The first necessary step to pay by check is to activate the setting. To do so, go to
:menuselection:`Accounting app --> Configuration --> Checks`, there you can activate the payment
method as well as set up the :guilabel:`Check Layout`.
Once you have activated the setting, you must also activate the payment method on related bank
journals. Go to :menuselection:`Accounting dashboard --> Bank Journal --> drop-down menu (⋮) -->
Configuration`.
In the :guilabel:`Outgoing Payments` tab, add :guilabel:`Checks` as a payment method.
```

File: content/applications/finance/accounting/payables/pay/check.rst [hunk 3]
```diff
@@ -2,39 +2,28 @@
...
 
-.. image:: check/check01.png
-  :align: center
+.. note::
+   Some countries require specific modules to print checks, such modules may be installed by
+   default. For instance the :guilabel:`U.S. Check Printing` module is required to print U.S.
+   checks.
```
Comment (jaae-odoo) [inline content/applications/finance/accounting/payables/pay/check.rst:26]:
```suggestion
.. note::
   Some countries require specific modules to print checks; such modules may be installed by
   default. For instance the :guilabel:`U.S. Check Printing` module is required to print U.S.
   checks.
```

File: content/applications/finance/accounting/payables/pay/check.rst [hunk 4]
```diff
@@ -48,76 +37,28 @@ For the United States, Odoo supports by default the check formats of:
...
-
-.. image:: check/check04.png
-  :align: center
-
-Once your payment by check is registered, don't forget to **Confirm** it.
-Once confirmed, you can use **Print Check** directly or follow the preceding
-flow to print checks in batch:
+On your :guilabel:`Accounting Dashboard` in the :guilabel:`Bank Journal`, you are able to see the
+number of checks registered. By clicking on :guilabel:`Checks to print` you have get the possibility
+to print the checks reconciled.
```
Comment (jaae-odoo) [inline content/applications/finance/accounting/payables/pay/check.rst:62]:
```suggestion
On your :guilabel:`Accounting Dashboard` in the :guilabel:`Bank Journal`, you can see the
number of checks registered. By clicking on :guilabel:`Checks to print` you have got the possibility
to print the reconciled checks.
```

**Review by jaae-odoo — CHANGES_REQUESTED** [review]
Hey, just a few minor typos, nothing else to add :)

**Review by jaae-odoo — APPROVED** [review]

**Review by jaae-odoo — APPROVED** [review]

**Review by xpl-odoo — APPROVED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**